fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -4193,9 +4193,21 @@ public void truncateTable(TruncateTableStmt truncateTableStmt) throws DdlExcepti
             TruncateTableInfo info = new TruncateTableInfo(db.getId(), olapTable.getId(), newPartitions,
                     truncateEntireTable);
             editLog.logTruncateTable(info);
+
+            // refresh mv
+            Set<Long> relatedMvs = olapTable.getRelatedMaterializedViews();
+            for (long mvId : relatedMvs) {
+                MaterializedView materializedView = (MaterializedView) db.getTable(mvId);
```

Comment:
can materializedView be null?

Reply:
I think mv can not be null here because we use write lock wrap this logic